approved/v93k/flow_control.tf

```diff
@@ -2413,12 +2455,12 @@ test_flow
   run_and_branch(pgm_3_BEA7F3B)
   then
   {
-    @T8_BEA7F3B_RAN = 1;
   }
   else
   {
-    @T8_BEA7F3B_RAN = 1;
+    stop_bin "", "fail", , bad, noreprobe, red, 10, over_on;
   }
+  @T8_BEA7F3B_RAN = 1;
```

Comment:
All cases of the new if_ran implementation where it doesn't affect the target test and now sets a flag after it has executed.